uxarray/helpers.py

```diff
@@ -673,3 +673,53 @@ def close_face_nodes(Mesh2_face_nodes, nMesh2_face, nMaxMesh2_face_nodes):
     np.put(closed.ravel(), first_fv_idx_1d, first_node_value)
 
     return closed
+
+
+def _convert_face_node_conn_to_sparse_matrix(
+        dense_matrix: np.ndarray, fill_value=INT_FILL_VALUE) -> tuple:
+    """Converts a given dense matrix connectivity to a sparse matrix format
+    where the locations of non-zero entries are stored using COO (coordinate
```

Comment:
Throughout this function, the sparse matrix format contains the location of **non fill-value entries**, not non zero locations.

Reply:
I think that's what we want here. Why do we want the non-zero locations (what does the non-zero location mean?)?

Reply:
Im referring to the name of the variable and the doc string.

Reply:
ohh Got it. Thanks for pointing it out